oprheum-game/InstructionOverlayNode.swift

```diff
@@ -36,35 +38,51 @@ class InstructionOverlayNode: SKNode {
             self.addChild(label2)
         }
     }
+   
+    var text3 = "" {
+        didSet(oldText) {
+            label3.removeFromParent()
+            label3 = makeLabel(text: text3)
+            label3.position = centeredPoint(at: label3Height)
+            self.addChild(label3)  
+        }
+    }
 
     private var background: SKShapeNode!
     private var label1: SKLabelNode!
     private var label2: SKLabelNode!
+    private var label3: SKLabelNode!
     private var labelContinue: SKLabelNode!
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    init(scene: BaseScene) {
-        size = scene.frame.size
+    convenience init(scene: BaseScene) {
+        self.init(size: scene.frame.size)
+    }
+    init(size: CGSize) {
+        self.size = size
         super.init()
 
-        background = SKShapeNode(path: backgroundPath())
-        background.fillColor = SKColor.blackColor()
-        background.alpha = 0.25
-        self.addChild(background)
+//        background = SKShapeNode(path: backgroundPath())
+//        background.fillColor = SKColor.blackColor()
+//        background.alpha = 0.25
+//        self.addChild(background)
```

Comment:
This will effect all overlays, including the instructions at the beginning of the game. Are you sure this is what you want to do?